pybind/CMakeLists.txt

```diff
@@ -1,16 +1,18 @@
-pybind11_add_module(pywallet3 MODULE
-  module.cpp
-  wallet/daemon_comms_config.cpp
-  wallet/rpc_config.cpp
-  wallet/keyring.cpp
-  wallet/keyring_manager.cpp
-  wallet/wallet.cpp
-  wallet/wallet_config.cpp
-)
-target_link_libraries(pywallet3 PUBLIC wallet3)
-target_include_directories(pywallet3 PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+if(BUILD_PYBIND)
```

Comment:
Rather than change this I think we should change the root cmake to put `if(BUILD_PYBIND)` around the `add_subdirectory(pybind)`